src/main/java/be/seeseemelk/mockbukkit/WorldMock.java

```
@@ -1688,6 +1903,12 @@ public Biome getBiome(int x, int y, int z)
 		throw new UnimplementedOperationException();
 	}
 
+	@Override
+	public @NotNull Biome getComputedBiome(int x, int y, int z)
+	{
+		return null;
```

Should throw an exception